app/connectors/CallEligibilityApiRequest.scala

```diff
@@ -16,17 +16,51 @@
 
 package connectors
 
-import play.api.libs.json.{Json, OFormat}
+import essttp.rootmodel.ttp.eligibility.{IdType, IdValue, Identification}
+import models.EligibilityReqIdentificationFlag
+import play.api.libs.json._
+import play.api.libs.functional.syntax._
 
 final case class CallEligibilityApiRequest(
     channelIdentifier:         String,
-    idType:                    String,
-    idValue:                   String,
+    identification:            List[Identification],
```

Comment:
does the stub need updating as well to be able to handle both forms of the request?